exercises/concept/temperature/.docs/hints.md

```diff
@@ -0,0 +1,11 @@
+# Temperature Exercise
```

Comment:
The headings are stripped when this content is used, so we usually just have it be the type of file

```suggestion
# Hints
```

Reply:
Do they (headings) serve any purpose in the hints file then?